src/components/AboutUsSection.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import Typography from "./Typography";
```

Comment:
cara importnya seperti ini

kalau lihat di preview vercelnya gambarnya tidak bisa ke baca kemungkinan karena manggilnya dari src danseterusny

https://kelompok-1b-git-about-page-ignasiusadhitias-projects.vercel.app/about

```suggestion
import React from "react";
import Typography from "./Typography";
import image from '../assets/AssetsAboutPages/Mask group.png' // cara manggil gambarnya seperti ini
```